### DISCUSSION TITLE
How to display my portfolio of metadata work on GitHub

### DISCUSSION THREAD

**Discussion by hdahlstrom94** [community/community#126454] [New to GitHub]
For context, I am a historian that focuses on the creation of metadata for the various types of collections I preserve. I want to use GitHub as a way to show off my portfolio of metadata that I have done in previous positions, but I don't know where to start or how to go about it. I did see the article of someone posting links of how librarians can use GitHub, but I am more of an example person and felt like the literature didn't help me. How would I go about this?

**Comment by namdng09** [comment]
Hi there! i guess you want to create something like this:
![image](https://github.com/community/community/assets/144111926/9e511468-a054-4fa2-b26c-7f06f8fac8ab)

Below these is step-by-step create portfolio with github:
1) Create new repository.
![image](https://github.com/community/community/assets/144111926/c06cafa8-12ae-49c8-a2da-f28b35ccd280)
- The name of repo is up to you, and set it public.

2) Enable Github Pages in the setting of the repo.
![image](https://github.com/community/community/assets/144111926/80e655e4-693e-4246-981d-f99c1adedb71)
- Go to setting of your repository
- In Code and automation, you click Pages
- Set branch to main (because it's the default branch)
- Click SAVE (maybe in there you can custom your url domain)

3) Clone this repo to your local machine.
![image](https://github.com/community/community/assets/144111926/77506f98-5695-451a-a3b5-219cf51f69f1)
- Go to the <> Code
- Click "Open with Github desktop" (the easiest way and you need install Github desktop)

4) Create HTML, CSS for your portfolio.
- You complete your code and push to Github 
- You also push code to Github by Github desktop

5) Finally, Get link and share your portfolio.
![image](https://github.com/community/community/assets/144111926/646b3b54-dfe5-4f34-95d5-6747e13905fa)

- You move back to Pages / Code and automation in Setting of repo
- Get you link!

Thankyou for reading, i hope you like it <3
Have a good day!
